ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/DeleteAppDialog.java

```diff
@@ -0,0 +1,58 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.view.dialogs;
+
+import org.iplantc.de.admin.desktop.client.toolAdmin.ToolAdminView;
+import org.iplantc.de.admin.desktop.client.toolAdmin.view.subviews.ToolPublicAppListWindow;
+import org.iplantc.de.client.models.IsHideable;
+import org.iplantc.de.client.models.apps.AppAutoBeanFactory;
+import org.iplantc.de.client.models.apps.AppList;
+import org.iplantc.de.commons.client.views.dialogs.IPlantDialog;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.inject.Inject;
+import com.google.web.bindery.autobean.shared.AutoBeanCodex;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.widget.core.client.container.FlowLayoutContainer;
+
+/**
+ * @author aramsey
+ */
+public class DeleteAppDialog extends IPlantDialog implements IsHideable {
+
+    private AppAutoBeanFactory appAutoBeanFactory = GWT.create(AppAutoBeanFactory.class);
+    private FlowLayoutContainer container;
+    @Inject ToolPublicAppListWindow publicAppListWindow;
+    @Inject ToolAdminView.ToolAdminViewAppearance appearance;
+
+    @Inject
+    public DeleteAppDialog(ToolAdminView.ToolAdminViewAppearance appearance, ToolPublicAppListWindow publicAppListWindow) {
```

Comment:
This dialog should be named `DeleteToolDialog`.
I also doesn't need both OK and Cancel buttons, since it doesn't execute any actions. It could just have an OK button.